src/backend/config/settings.py

```diff
@@ -10,21 +10,34 @@
 )
 
 # In order to get the env vars from the top level every model need to inherit from BaseSettings with this config
-setting_config = SettingsConfigDict(
+SETTINGS_CONFIG = SettingsConfigDict(
     env_file=".env",
     extra="ignore",
     env_nested_delimiter="_",
     env_prefix="",
     env_ignore_empty=True,
 )
 
+CONFIG_PATH = "src/backend/config"
```

Comment:
Small refactor to just convert these constants into caps, in line with the rest of the codebase